script/start

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+#
+# Run single command from Procfile
+#
+if [ ! "$1" ]; then
+  echo "Usage: $0 <process type>" 1>&2
+  exit 1
+fi
+
+PROCFILE=`dirname "$0"`/../Procfile
+CMD=`cat "$PROCFILE" | grep "^$1:" | cut -d: -f2-`
+if [ ! "$CMD" ]; then
+  echo "Process name $1 not found in Procfile" 1>&2
+  exit 2
+fi
+
+PS="$1"
+[ "$PORT" ] || PORT=3000
```

Comment:
do not set environment variibales in different places. `Dockerfile` should be the only place for that.

Reply:
This is meant to be a small standalone tool resembling [foreman](https://ddollar.github.io/foreman/) but running only a single process (even though foreman uses port 5000 by default). Maybe I should explain that a bit more in the comments. Since `$PORT` is almost always used in the `Procfile`, one needs to have it defined.
I'd love to use an existing tool for this, but didn't find it.

Reply:
can we put a `ENV PORT=3000` into the `Dockerfile` instead?
What's the purpose of `PS`?

Reply:
I think it would be useful to be able to run this script also outside a docker container (if you install Foodsoft in a different way). For this case, there is a default port. But I have nothing against giving an error if PORT is not set, would that be ok?

Reply:
`PS` is the worker/dyno type, it can be used in the `Procfile` if one wants.

Reply:
i don't see any usage of it ATM. i'd keep the script as simple as possible, without any unused feature. IMHO it should only do the minimal stuff to get OUR Procfile running and the rest is up to any other service provider

Reply:
Ok, I'll leave out `PS` and show an error message if `PORT` is unset.